src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -98,6 +104,124 @@ public static function getExtras()
         return $extras;
     }
 
+    public static function getSystemStatus(): array
+    {
+        $systemStatus = [];
+        $systemStatus['order'] = self::getOrderSystemStatus();
+        $systemStatus['task_processor'] = self::getTaskProcessorStatus();
+        $systemStatus['failed_compatibility_checks'] = self::getFailedCompatibilityChecks();
+
+        return $systemStatus;
+    }
+
+    public static function getFailedCompatibilityChecks(): array
+    {
+        return array_merge(
+            PluginConflictChecker::getPluginsWithConflict(),
+            ServerStatusChecker::getServerIssues(),
+        );
+    }
+
+    public static function getTaskProcessorStatus(): array
+    {
+        $taskProcessorStatus = [];
+
+        $now = current_time('mysql', 1);
```

Comment:
you some times use ` current_datetime()` and sometimes `current_time('mysql', 1)` i think we should use only one

Reply:
Changes for date uniformity implemented.